custom/icds_reports/static/icds_reports/js/spec/adolescent-girls.directive.spec.js

```diff
@@ -0,0 +1,96 @@
+"use strict";
+
+var pageData = hqImport('hqwebapp/js/initial_page_data');
+
+
+describe('AdolescentGirlsDirective', function () {
+
+    var $scope, $httpBackend, $location, controller;
+
+    pageData.registerUrl('icds-ng-template', 'template');
+    pageData.registerUrl('adolescent_girls', 'adolescent_girls');
+    pageData.registerUrl('icds_locations', 'icds_locations');
+
+    beforeEach(module('icdsApp', function($provide) {
+        $provide.constant("userLocationId", null);
+    }));
+
+    beforeEach(inject(function($rootScope, $compile, _$httpBackend_, _$location_) {
+        $scope = $rootScope.$new();
+        $httpBackend = _$httpBackend_;
+        $location = _$location_;
+        $httpBackend.expectGET('template').respond(200, '<div></div>');
+        $httpBackend.expectGET('adolescent_girls').respond(200, {
+            report_data: [],
```

Comment:
It looks like this is causing this to always return an empty set. Will there be more tests that expect real looking data being returned?